### PR TITLE
Bugfix in reportWaterUsage function

### PR DESCRIPTION
**Pull request by vartika271987**
Tested and ready to fix now.

### CODE REVIEW

File: R/reportWaterUsage.R [hunk 1]
```diff
@@ -17,19 +17,19 @@
 
 reportWaterUsage<-function(gdx, detail=TRUE) {
   
-  out <- water_usage(gdx,level="regglo",users=NULL,sum=FALSE,digits=3)[,,"agriculture"]
-  getNames(out) <- "Resources|Water|Withdrawal|Agriculture (km3/yr)"
-   
+  x <- water_usage(gdx, level="regglo",users=NULL, digits=3)[,,"agriculture"]
```
Comment (mishkos) [inline R/reportWaterUsage.R:20]: I'd keep `sum=FALSE` even though it's a default parameter. Just for the clarity of the code.

File: R/reportWaterUsage.R [hunk 2]
```diff
@@ -17,19 +17,19 @@
...
+  getNames(x) <- "Resources|Water|Withdrawal|Agriculture (km3/yr)"
+  
   if (detail==TRUE) {
-    y <- water_usage(gdx,level="regglo",users="kcr",sum=FALSE,digits=3)[,,"agriculture"]
+    y <- water_usage(gdx,level="regglo",users="kcr",sum=TRUE,digits=3)
```
Comment (mishkos) [inline R/reportWaterUsage.R:24]: Why `sum=TRUE` here? Is there an error coming when running magpie with different realization of `water_demand` module? One realization deals with all sectors and another one only with agricultture. In that case we have to change the `water_usage` function to account for all cases in the module.

Comment (vartika271987) [reply R/reportWaterUsage.R:24]: sum=FALSE, in this case, does not sum over all crops in the crops category, and the same for livestock. It reports water use for every crop and livestock category. Therefore, sum=TRUE, In this case, sums over the respective crop and livestock categories to report total water use.

File: R/reportWaterUsage.R (hunk 1 above)
Comment (vartika271987) [reply R/reportWaterUsage.R]: Updated

File: R/reportWaterUsage.R (hunk 2 above)
Comment (mishkos) [reply R/reportWaterUsage.R:24]: isn't `sum` in the `water_use` function to sum over sectors? At least the documentation says so. You write as if it's over products.

**Review by mishkos — APPROVED** [review]

**Pull request merged**